lua/telescope/builtin/lsp.lua

```diff
@@ -120,26 +120,32 @@ lsp.code_actions = function(opts)
     return
   end
 
-  local _, response = next(results_lsp)
-  if not response then
-    print("No code actions available")
-    return
+  local idx = 1
+  local all_results = {}
+
+  for _, response in pairs(results_lsp) do
+    local results = response.result
+    if not results or #results == 0 then
+      goto continue
```

Comment:
we should avoid `goto` :)

Reply:
:+1:, wanted to avoid nested code. I guess the only other option is negating the if condition and move stuff inside it?

Reply:
No goto because of lua 5.1 compatibility.